src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public static void doWork() {
+        System.out.println("Bulldozer started its work");
```

Comment:
The message 'Bulldozer started its work' is not very informative. It would be better if the message indicated what Bulldozer is doing, for example, 'Bulldozer is moving earth'.